src/pf-alert/pf-alert.component.js

```diff
@@ -1,5 +1,5 @@
-import {default as tmpl} from 'pf-alert.template';
-import {default as utils} from 'pf-utils';
+import {default as tmpl} from './pf-alert.template';
+import {default as utils} from '../pf-utils/pf-utils';
```

Comment:
This is what I'm trying to avoid. I don't believe we should be hard coding relative paths here. Like I tried to suggest above, if another component requires pf-alert (e.g., in another repo), this won't build unless the directory structure is exactly the same.